### ISSUE TITLE
V4

### ISSUE THREAD

**Issue by kinmami** [SukkoPera/LittleSixteen#15]
U91~95 lengthen the pad footprint and add a DOT on pin 1

**Comment by kinmami** [comment]
CT1 enlarge the pad holes

**Comment by kinmami** [comment]
CN6 pinout is wrong

**Comment by kinmami** [comment]
to replace power resistors R91, R92, R94, R95 and R39, R40 with polyfuses (BOM list updated with the commercial p/n)

**Comment by kinmami** [comment]
R32 for MOS 6581: to align the schematic and the interactive BOM list

**Comment by gordonfpanam** [comment]
> R32 for MOS 6581: to align the schematic and the interactive BOM list

MOS 6581 as in a SID chip? Is this an attempt to integrate the SID for Psytronix TED titles that support SID expansion carts?

**Comment by SukkoPera** [comment]
Yes, V4 has a built-in SIDcard. See https://plus4world.powweb.com/forum/47104.

**Comment by gordonfpanam** [comment]
I'll be looking for the PCBWay link to order v4 boards as soon as you have it up.

**Comment by SukkoPera** [comment]
@gordonfpanam: That's coming in a few days, but if you want I can send you the gerbers and you can order the board on JLC, which should be cheaper.
